examples/triton/onnx/README.md

```diff
@@ -138,13 +138,18 @@ Inside the container shell, there are two options to install BentoML:
 - Install from editable
 
 ```bash
-cd /opt/bentoml && pip install -r requirements/dev-requirements.txt
+curl -sSL https://raw.githubusercontent.com/pdm-project/pdm/main/install-pdm.py | pdm run -v python -
+export VIRTUAL_ENV=$(dirname $(dirname $(which python)))
+pushd /opt/bentoml 2>/dev/null
+~/.local/bin/pdm venv create --with-pip
+~/.local/bin/pdm install -dG testing -G tooling -G tracing -G grpc -G io -G aws
+popd 2>/dev/null
 ```
 
 Run the [`setup` script](./setup):
 
 ```bash
-cd /workspace/ && pip install -r requirements/requirements.txt
+pushd /workspace/ && ~/.local/bin/pdm run pip install -r requirements/requirements.txt
```

Comment:
I'd prefer not introduce PDM in examples to not burdening users
If we just want to install bento, use `pip install -e` with necessary dependency groups

Reply:
got it